CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.6] - 2021-06-24
+### Fixed
+- Fix some javascript errors.
```

Comment:
Can you explain better what javascript code did you fixed?

Reply:
The error was: `cannot read property 'enableLogs' of undefined`

Reply:
The error specifically was this:
![image](https://user-images.githubusercontent.com/24899146/123594510-176bdd80-d7f0-11eb-99f2-3f5a0d87abfc.png)